.github/workflows/bundle-release.yml

```diff
@@ -56,7 +56,7 @@ jobs:
 
       - name: Install operator-sdk
         run: |
-          RELEASE_VERSION=v1.14.0
+          RELEASE_VERSION=v1.29.0
```

Comment:
this version will cause an error while run `make redhat-certificated-bundle`

```
Generating kustomize files in config/manifests
-: pattern /home/runner/work/function-mesh/function-mesh/api/...: main module (github.com/streamnative/function-mesh) does not contain package github.com/streamnative/function-mesh/api/compute/v1alpha1
FATA[0000] Error generating kustomize files: error getting ClusterServiceVersion base: error generating ClusterServiceVersion definitions metadata: one or more API packages had type errors
```

Reply:
any error from `make bundle`?

Reply:
it has same error if use v1.14.0, all caused by command:
```
operator-sdk generate kustomize manifests -q
```

Reply:
I cannot reproduce it, could you please delete the `bundle` directory and try again to see if any error occurs?

Reply:
yes, it's same error, did you run the command with this PR or on master branch?